Description: Disable Sonar features since it is not available in Debian yet
Author: Miguel Landaeta <miguel@miguel.cc>
Author: Damien Raude-Morvan <drazzib@debian.org
Forwarded: not-needed
Last-Update: 2012-08-18

--- a/build.gradle
+++ b/build.gradle
@@ -87,7 +87,7 @@
     publishedProjects = [project(':core'), project(':toolingApi'), project(':wrapper'), project(':baseServices'), project(':messaging')]
     pluginProjects = [
         'plugins', 'codeQuality', 'jetty', 'antlr', 'wrapper', 'osgi', 'maven',
-        'ide', 'announce', 'scala', 'sonar', 'signing', 'cpp', 'ear', 'javascript', 'buildComparison',
+        'ide', 'announce', 'scala', 'signing', 'cpp', 'ear', 'javascript', 'buildComparison',
         'diagnostics', 'reporting', 'publish', 'ivy'
     ].collect {
         project(it)
--- a/settings.gradle
+++ b/settings.gradle
@@ -36,7 +36,7 @@
 include 'toolingApi'
 include 'docs'
 include 'integTest'
-include 'sonar'
+//include 'sonar'
 include 'signing'
 include 'cpp'
 include 'ear'
--- a/subprojects/docs/src/docs/dsl/dsl.xml
+++ b/subprojects/docs/src/docs/dsl/dsl.xml
@@ -339,9 +339,11 @@
             <tr>
                 <td>org.gradle.plugins.signing.Sign</td>
             </tr>
+<!--
             <tr>
                 <td>org.gradle.api.plugins.sonar.SonarAnalyze</td>
             </tr>
+-->
             <tr>
                 <td>org.gradle.api.tasks.Sync</td>
             </tr>
@@ -448,4 +450,4 @@
         </table>
     </section>
 
-</book>
\ No newline at end of file
+</book>
--- a/subprojects/docs/src/docs/userguide/sonarPlugin.xml
+++ b/subprojects/docs/src/docs/userguide/sonarPlugin.xml
@@ -223,9 +223,8 @@
             <tr>
                 <td><literal>sonarAnalyze</literal></td>
                 <td>-</td>
-                <td><apilink class="org.gradle.api.plugins.sonar.SonarAnalyze"/></td>
                 <td>Analyzes a project hierarchy and stores the results in the Sonar database.</td>
             </tr>
         </table>
     </section>
-</chapter>
\ No newline at end of file
+</chapter>
--- a/subprojects/docs/src/docs/userguide/userguide.xml
+++ b/subprojects/docs/src/docs/userguide/userguide.xml
@@ -60,8 +60,10 @@
     <xi:include href='findBugsPlugin.xml'/>
     <xi:include href='jdependPlugin.xml'/>
     <xi:include href='pmdPlugin.xml'/>
+<!--
     <xi:include href='sonarPlugin.xml'/>
     <xi:include href='sonarRunnerPlugin.xml'/>
+-->
     <xi:include href='osgi.xml'/>
     <xi:include href='eclipsePlugin.xml'/>
     <xi:include href='ideaPlugin.xml'/>
